js/index.js

```diff
@@ -63,6 +63,54 @@ function displayProjects() {
   projectCards.appendChild(cards);
 }
 
+const handleSubmit = (event) => {
+  event.preventDefault();
+  const nameError = document.getElementById('name-error');
+  const emailError = document.getElementById('email-error');
+  const emailIcnorect = document.getElementById('emile-incorect');
+  const msgError = document.getElementById('msg-error');
+  const name = document.getElementById('name-input').value;
+  const email = document.getElementById('email-input').value;
+  const msg = document.getElementById('msg-input').value;
+  let error = false;
+
+  if (name.length === 0) {
+    nameError.innerHTML = 'Name is required';
+    error = true;
+  }
+  if (error === false) {
+    if (email.length === 0) {
+      emailError.innerHTML = 'Email is required';
+      error = true;
+    }
+    nameError.innerHTML = '';
+  }
+  if (error === false) {
+    if (!email.match('[a-z0-9]+@[a-z]+.[a-z]{2,3}')) {
+      emailIcnorect.innerHTML = 'Email is incorect';
+      error = true;
+    }
+    emailError.innerHTML = '';
+  }
+  if (error === false) {
+    if (email !== email.toLowerCase()) {
+      emailIcnorect.innerHTML = 'Email is incorect';
+      error = true;
+    }
```

Comment:
**[OPTIONAL]** Nice job implementing the changes requested by the previous reviewers and I hight recommend you fix the typo errors in lines 90 and 97 where it says `emailIcnorect` to make your code base professional also one more thing try to give a more descriptive error message in line those two lines rather than `Email is incorrect` (also fix the typo here too) make it something like `Uppercase email is not allowed` or something similar like that to keep professional code base.